src/utils/createRowsSelector.js

```diff
@@ -0,0 +1,71 @@
+import {chunk, map, values} from 'lodash';
```

Comment:
Could you import lodash methods individually to reduce bundle size? I.e in the form:

`import chunk from 'lodash/chunk';`